ocs_ci/framework/conf/ocp_version/ocp-4.11-ga-config.yaml

```diff
@@ -0,0 +1,17 @@
+---
+# Config file for OCP GA 4.11 stable channel
+
+RUN:
+  client_version: '4.11-ga'
+
+DEPLOYMENT:
+  ocp_url_template: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/{file_name}-{os_type}-{version}.tar.gz"
+  installer_version: "4.11-ga"
+  terraform_version: "0.12.26"
+  # ignition_version can be found here
+  # https://docs.openshift.com/container-platform/4.10/post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks
+  ignition_version: "3.2.0"
+  ocp_channel: "stable"
+ENV_DATA:
+  vm_template: "rhcos-4.10.3-x86_64-vmware.x86_64"
```

Comment:
We still don't have the templates?  Can you please ask in aos-devel?

Reply:
this files shouldn't be here. somehow my branch was messed up. raised https://github.com/red-hat-storage/ocs-ci/pull/6337 and closing this one.

not yet. already sent mail for the same.